README.md

```diff
@@ -89,7 +96,7 @@ We can see the timestamp transaction using getrawblockchain command
 }
 ```
 
-### Rails support
+#### Rails support
 
 Gruby supports ruby on rails integration.
```

Comment:
Would you fix this typo if possible?
Gruby -> Glueby

Reply:
Fixed at 3d990dc9308481e8250f53a2cc6a1ff916ed69c9